vault/resource_ldap_secret_backend_static_role_test.go

```diff
@@ -43,6 +43,13 @@ func TestAccLDAPSecretBackendStaticRole(t *testing.T) {
 					resource.TestCheckResourceAttr(resourceName, consts.FieldRotationPeriod, rotationPeriod),
 				),
 			},
+			{
+				SkipFunc: func() (bool, error) {
+					return !provider.IsAPISupported(testProvider.Meta(), provider.VaultVersion116), nil
```

Comment:
We should use the `testProvider` for `ProviderMeta` related bits in test code. Here's [an example](https://github.com/hashicorp/terraform-provider-vault/blob/main/vault/resource_namespace_test.go#L97-L99) of this being done (I broke it into multiple lines below for easier readability for now, 1 line is also fine if you'd prefer that)

```suggestion
                                          meta := testProvider.Meta().(*provider.ProviderMeta)
					return !meta.IsAPISupported(provider.VaultVersion116), nil
```

Reply:
Disregard, I misread the code. Apologies!